packages/code-studio/src/settings/ColumnSpecificSectionContent.tsx

```diff
@@ -614,10 +614,9 @@ export class ColumnSpecificSectionContent extends PureComponent<
 
     return (
       <div className="app-settings-formatting-section" ref={this.containerRef}>
-        <div>Default formatting for matched column names</div>
         <div className="app-settings-menu-description mb-3">
-          Applies a formatting rule to all columns that match a specified name
-          and type.
+          Customize the formatting of table columns by applying rules based on
+          matching both a column name and column type.
```

Comment:
```suggestion
          Customize the formatting of table columns by applying rules based on
          name and type.
```

Reply:
That's shorter but not as clear that it matches column names and not table names, and that the requirement to match is on both, which "and" implies. But it could be mis-interpreted to mean you can have rules matching name and rules matching type.

We have space to be more clear in this case.